src/components/TeamMemberTasks/TeamMemberTasks.jsx

```diff
@@ -87,174 +87,183 @@ const TeamMemberTasks = props => {
 
       // const teamMembers = [];
 
-      // fetch all team members for each team
-      managingTeams.forEach(team => {
-        teamMembersPromises.push(httpService.get(ENDPOINTS.TEAM_MEMBERS(team._id)));
-      });
-
-      Promise.all(teamMembersPromises).then(data => {
-        // console.log('team members', data);
-        for (let i = 0; i < managingTeams.length; i++) {
-          allManagingTeams[i] = {
-            ...managingTeams[i],
-            members: data[i].data,
-          };
-          allMembers = allMembers.concat(data[i].data);
-        }
-
-        // fetch all time entries for current week for all members
-        const uniqueMembers = _.uniqBy(allMembers, '_id');
-
-        // console.log('members: ', uniqueMembers);
-
-        const membersId = [];
-        for (let i = 0; i < uniqueMembers.length; i++) {
-          membersId.push(uniqueMembers[i]._id);
-        }
-
-        if (membersId.length) {
-          memberTimeEntriesPromises.push(
-            httpService.get(ENDPOINTS.TIME_ENTRIES_USER_LIST(membersId)).catch(err => {}),
-          );
-        }
-
-        if (memberTimeEntriesPromises.length) {
-          Promise.all(memberTimeEntriesPromises).then(data => {
-            // console.log('time entries: ', data);
-            if (data[0]?.data.length === 0) {
-              for (let i = 0; i < uniqueMembers.length; i++) {
-                uniqueMembers[i] = {
-                  ...uniqueMembers[i],
-                  timeEntries: [],
-                };
-              }
-            } else {
-              for (let i = 0; i < uniqueMembers.length; i++) {
-                const entries = [];
-                for (let j = 0; j < data[0]?.data.length; j++) {
-                  if (uniqueMembers[i]._id === data[0].data[j].personId) {
-                    entries.push(data[0].data[j]);
-                    // console.log('push');
-                  }
-                }
-                uniqueMembers[i] = {
-                  ...uniqueMembers[i],
-                  timeEntries: entries,
-                };
-              }
-            }
-            // console.log('members after entries: ', uniqueMembers);
-
-            // fetch all tasks for each member
-            teamMemberTasksPromises.push(
-              httpService.get(ENDPOINTS.TASKS_BY_USERID(membersId)).catch(err => {
-                if (err.status !== 401) {
-                  console.log(err);
-                }
-              }),
+      console.log(props);
```

Comment:
This one I didn't review but click on the `Viewed` checkbox because it's under changing by management-dashboard team and it will be updated soon.

Reply:
Alright, should I just leave it like this then? And then once this gets merged they can work on top of it

Reply:
Feel free to keep your changes here